<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EcuDialog</class>
 <widget class="QDialog" name="EcuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECU Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/png/org.genivi.DLTViewer.png</normaloff>:/icons/png/org.genivi.DLTViewer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxUpdate">
     <property name="text">
      <string>Update data in DLT if online</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ECU</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0">
        <widget class="QComboBox" name="comboBoxInterface">
         <item>
          <property name="text">
           <string>TCP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UDP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Serial</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Default Trace Status:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QCheckBox" name="checkBoxTiming">
         <property name="text">
          <string>Timing packets from ECU</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QComboBox" name="loglevelComboBox">
         <item>
          <property name="text">
           <string>off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fatal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>warn</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>verbose</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Default Log Level:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QComboBox" name="tracestatusComboBox">
         <item>
          <property name="text">
           <string>off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>on</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="descriptionLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Interface Type:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Verbose Mode:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QCheckBox" name="checkBoxGetLogInfo">
         <property name="text">
          <string>Send &quot;Get Log Info&quot; if online</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="idLineEdit">
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QCheckBox" name="checkBoxDefaultLogLevel">
         <property name="text">
          <string>Send &quot;Default Log Level,Trace Status and Verbose Mode&quot; if online</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QComboBox" name="comboBoxVerboseMode">
         <item>
          <property name="text">
           <string>Non-Verbose Mode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Verbose Mode</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QCheckBox" name="checkBoxGetSoftwareVersion">
         <property name="text">
          <string>Send &quot;Get SW Version&quot; if online</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <attribute name="title">
       <string>TCP</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="comboBoxHostname">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="comboBoxPortIP">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinBoxAutoreconnect">
         <property name="toolTip">
          <string>Automatic reconnect after specified timeout  if no data from ECU was received.</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>IP Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Hostname:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBoxAutoReconnect">
         <property name="toolTip">
          <string>Automatic reconnect after specified timeout  if no data from ECU was received.</string>
         </property>
         <property name="text">
          <string>Auto reconnect timeout in seconds</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_UDP">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>UDP</string>
      </attribute>
      <widget class="QLabel" name="label_UDP_receiver">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>194</width>
         <height>14</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Receiving interface:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_UDP_mcastadress">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>194</width>
         <height>14</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Multicast address:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxUDP_mcastaddress">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxNetworkIF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxMulticast">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>194</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Multicast on network interface</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxPortIP_UDP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>194</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>IP Port:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <attribute name="title">
       <string>Serial</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Baudrate:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="comboBoxBaudrate">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxPortSerial">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>COM0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyUSB1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyS0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyS1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxSendSerialHeaderSerial">
         <property name="text">
          <string>Send Serial Header</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBoxSyncToSerialHeaderSerial">
         <property name="text">
          <string>Sync to Serial Header when receiving</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EcuDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EcuDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
